scripts/init_ladybug.py

```
@@ -227,8 +257,24 @@ def init_ladybug(db_path: Path, dry_run: bool = False) -> None:
                 print(f"  ⚠️  {desc}: {exc}")
 
+    # ── Column migrations ──────────────────────────────────────────────
+    print("\nColumn migrations (MemoryNode):")
+    for col, typ, default in MEMORYNODE_COLUMN_MIGRATIONS:
+        default_clause = f" {default}" if default else ""
+        cypher = f"ALTER TABLE MemoryNode ADD IF NOT EXISTS {col} {typ}{default_clause}"
+        try:
+            conn.execute(cypher)
+            print(f"  ✅ {col} {typ}{default_clause}")
+            changes += 1
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)